jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -12,32 +12,22 @@ pipeline {
         }
     }
     parameters {
-        string(
-            name: 'BUILD_ID',
-            description: 'Build ID of the distribution build manifest generated.',
-            trim: true
-        )
         string(
             name: 'VERSION',
-            description: 'Version of the distribution build manifest.',
+            description: 'Version of the release to download the artifacts.',
             trim: true
         )
     }
     environment {
-        BUILD_MANIFEST = "builds/opensearch/manifest.yml"
+        BUILD_MANIFEST = "opensearch-$VERSION/manifest.yml"
     }
     stages {
         stage('Create Release Tag') {
             steps {
                 script {
-                    manifest_path = "distribution-build-opensearch/${VERSION}/${BUILD_ID}/linux/x64/builds/opensearch/manifest.yml"
-                    downloadFromS3(
-                        destPath: "$WORKSPACE/$BUILD_MANIFEST",
-                        bucket: "${ARTIFACT_BUCKET_NAME}",
-                        path: "${manifest_path}",
-                        force: true
-                    )
-                    echo "Distribution build manifest is downloaded to $BUILD_MANIFEST"
+                    release_url = "https://artifacts.opensearch.org/releases/bundle/opensearch/$VERSION/opensearch-$VERSION-linux-x64.tar.gz"
```

Comment:
Yeah we are not uploading manifest.yml. If we want to it can be a part of https://github.com/opensearch-project/opensearch-build/blob/main/vars/promoteArtifacts.groovy but wondering if it is required since manifest is already bundled inside the tar.gz

Reply:
I think if someone wants the manifest without downloading the entire tar.gz, like here, it means we probably should. It's NBD.